sw/device/lib/testing/spi_flash_testutils.h

```diff
@@ -96,4 +166,35 @@ void spi_flash_testutils_program_page(dif_spi_host_t *spih, uint8_t *payload,
                                       size_t length, uint32_t address,
                                       bool addr_is_4b);
 
+/**
+ * Perform a read via the requested opcode.
+ *
+ * @param spih A SPI host handle.
+ * @param opcode The desired read opcode.
+ * @param payload A pointer to the buffer to receive data from the device.
+ * @param length Number of bytes in the buffer. Must be less than or equal to
+ *               256 bytes.
+ * @param address The start address where the read should begin.
+ * @param addr_is_4b True if `address` is 4 bytes long, else 3 bytes.
+ * @param width The width of the read (1, 2 or 4 bits).
+ * @param dummy The number of dummy cycles required after the address phase.
+ */
+void spi_flash_testutils_read_op(dif_spi_host_t *spih, uint8_t opcode,
+                                 uint8_t *payload, size_t length,
+                                 uint32_t address, bool addr_is_4b,
+                                 uint8_t width, uint8_t dummy);
+
+/**
+ * Enable or disable Quad mode on the EEPROM according to the SFDP-described
+ * method.
+ *
+ * @param spih A SPI host handle.
+ * @param method The method to use to enable.  This value should come from the
```

Comment:
Nit.
```suggestion
 * @param method The method to use to enable. This value should come from the
```

Reply:
https://www.thesaurus.com/e/writing/is-there-1-space-or-2-after-a-period/
"""
According to research, the preference for two spaces after a period falls along generational lines. Studies have shown that, beginning with millennials, younger generations widely prefer the single space after a period. On the flip side, Gen X and older generations usually prefer the double space.
"""

Am Gen-X.  I claim the exemption.